### PR TITLE
[TW-1404] API builder refresh

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Updated two screenshots

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/the-api-workflow.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ The following are the high-level steps involved when designing and developing AP
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:42]: Should we increase the `updated` date since this is a refresh ticket?

Comment (petewitcher-postman) [reply src/pages/docs/designing-and-developing-your-api/the-api-workflow.md:42]: Yes! Thanks for catching that.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! I added a question about the `updated` date in the front matter.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**